tests/pfcwd/conftest.py

```diff
@@ -67,31 +67,9 @@ def fake_storm(request, duthosts, enum_rand_one_per_hwsku_frontend_hostname):
     return request.config.getoption('--fake-storm') if not isMellanoxDevice(duthost) else False
 
 
-def update_t1_test_ports(duthost, mg_facts, test_ports, asic_index, tbinfo):
```

Comment:
I think we still need this function. There are plans to modify the t1-64-lag topo to have some inactive ports

Reply:
@neethajohn added back the function and make it run for all asic (as complete system)